### PR TITLE
BDOG-1252: Removes weighting system

### PR DESCRIPTION
**Pull request by Tyrpix**

### CODE REVIEW

File: app/uk/gov/hmrc/healthindicators/controllers/RepoScoreController.scala [hunk 1]
```diff
@@ -20,15 +20,15 @@ import javax.inject.{Inject, Singleton}
...
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 import scala.concurrent.ExecutionContext
 
 @Singleton
-class WeightedRepoScoreController @Inject()(
-                                            weightedRepoScorerService: WeightedRepoScorerService,
-                                            cc: ControllerComponents
+class RepoScoreController @Inject()(
+                                             weightedRepoScorerService: RepoScorerService,
```
Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/controllers/RepoScoreController.scala:30]: I think we can remove the weighted prefix from the parameter name.
Also, we can call `RepoScorerService ` as `RepoScorer`

File: app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala [hunk 2]
```diff
@@ -49,7 +49,7 @@ case class BobbyRulesRating(
 object BobbyRulesRating {
     def calculate(bobbyRulesRating: BobbyRulesRating): Int = {
```
Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:50]: The name of the method isn't helpful. I am not sure what it is calculating.

Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:50]: Can we not inline this method where it is used?

File: app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala [hunk 3]
```diff
@@ -49,7 +49,7 @@ case class BobbyRulesRating(
...
             val maxScore = 100
-            val activeViolationPenalty= 100
+            val activeViolationPenalty = 100
```
Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:52]: These constant values can live in the companion object directly as private members.

Comment (Tyrpix) [reply app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:52]: These will be replaced by a config file containing JSON, will be handled in another ticket where we are going to revise all these calculate methods for each rating type

File: app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala [hunk 4]
```diff
@@ -49,7 +49,7 @@ case class BobbyRulesRating(
...
             val pendingViolationPenalty = 20
 
             val score = maxScore-
```
Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:55]: Do we need the max score and difference? IIRC, we discussed we will just calculate the raw score based on penalty points and violations and return that number. Even if it is negative it is ok. The numbers will be aggregated later across other metrics and that will be the final score for that repo/service.

File: app/uk/gov/hmrc/healthindicators/controllers/RepoScoreController.scala (hunk 1 above)
Comment (Tyrpix) [reply app/uk/gov/hmrc/healthindicators/controllers/RepoScoreController.scala:30]: good spot thanks!

File: app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala (hunk 4 above)
Comment (Tyrpix) [reply app/uk/gov/hmrc/healthindicators/raters/bobbyrules/BobbyRulesRating.scala:55]: This is going to happen in another ticket, this was just to remove the weight system

File: app/uk/gov/hmrc/healthindicators/services/RepoScorerService.scala [hunk 5]
```diff
@@ -17,27 +17,31 @@
...
+      RepoScoreBreakdown(repo, sumScores(repoRating), repoRating.ratings)
     }))
   }
 
   def repoScoreAllRepos(): Future[Map[String, Int]] =
     for {
-      indicators <- repository.latestRatings()
-      scores = indicators.map(h => (h.repo, weightService.weightedScore(h))).toMap
+      ratings <- repository.latestRatings()
+      scores = ratings.map(h => (h.repo, sumScores(h))).toMap
```
Comment (anshulbajpai) [inline app/uk/gov/hmrc/healthindicators/services/RepoScorerService.scala:38]: There wasn't any test change required?

**Review by anshulbajpai — APPROVED** [review]

**Pull request merged**